chapters/logging.rst

```diff
@@ -0,0 +1,242 @@
+==============================
+Logging configuration in Jina
```

Comment:
**C**onfiguration 

We use title case for headers :)

Reply:
Sorry, I ignored this one. I thought it means to capitalize the first. Will keep that in mind. Thanks!